docs/source/reference/optimize.rst

```diff
@@ -0,0 +1,8 @@
+Automatic Kernel Parameters Optimizations
+=========================================
+
```

Comment:
Could you add some more information here?

* Optuna installation is required
* Currently it works for reduction ops only